account_tax_cash_basis_fix_currency_rate_difference/models/account_partial_reconcile.py

```diff
@@ -31,10 +31,10 @@ def _fix_multiple_exchange_rates_diff(
                     rec.credit_move_id.journal_id.type == 'purchase' else
                     rec.credit_move_id)
                 lines = []
+                invoice_taxes = invoice_move.move_id.line_ids.filtered(
+                    'tax_ids').mapped('tax_ids').filtered('use_cash_basis')
```

Comment:
what will happen if you have one line with IVA 16% and RET IVA 4%, then one line with only IVA 16%?

Reply:
It doesn't matter because the revaluation will be in a global context, i mean, I only need the taxes of the invoice (not for line) then, with the invoices taxes I compute the corresponding rate for each tax based on the invoice rate and the diff_amount :)